packages/svelte/src/internal/server/index.js

```diff
@@ -138,7 +144,10 @@ export function render(component, options = {}) {
 	return {
 		head,
 		html: payload.out,
-		body: payload.out
+		body: payload.out,
+		htmlAttributes: [...payload.htmlAttributes]
+			.map(([name, value]) => `${name}="${value}"`)
+			.join(' ')
```

Comment:
Someone might have an existing classname in their HTML template, in which case giving them a string would make it awkward to combine stuff. Should we return an object instead of a string, so that they have more flexibility?

Reply:
I briefly thought about that but it seemed unnecessary - in which case would you have existing attributes on a html tag but in such a way that you know which ones to then merge them in some way? Even if, the regex for adjusting the html attributes string would be straightforward. So I opted for making the simple case more ergonomic.

Reply:
it only really matters for new projects, I think, since we can't retroactively add `%htmlAttributes%` anyway. I think we just replace `lang="en"` with `%htmlAttributes%` in the template project's `app.html`, and add this to the root layout:

```svelte
<svelte:html lang="en" />
```